tumblr-photo-video-ripper.py

```diff
@@ -196,7 +198,9 @@ def _download_media(self, site, medium_type, start):
                 break
 
             try:
-                data = xmltodict.parse(response.content)
+                xml_cleaned = re.sub(u'[^\n\r\t\x20-\x7f]+',
```

Comment:
I was wondering whether we should ignore `\n`, `\r` and `\t`, since they are not the non-ASCII characters.

Reply:
While I debug the xml from tumblr, some attributes has '\n', '\r', '\t'. Hence, I add this 3 for avoid xml parser error. But your I would like to test whether is it ok without '\n', '\r', '\t'.